src/components/Collection/AnidbDescription.tsx

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
-import { mdiAccount } from '@mdi/js';
-import { Icon } from '@mdi/react';
 
-const charRegex = /(http:\/\/anidb\.net\/c(?:h|r)[0-9]+) \[([^\]]+)]/g;
+const RemoveSummaryRegex = /^\n(Source|Note|Summary):.*/mg;
 
-const AnidbLink = ({ character, text, url }) => (
-  <span className="text-panel-primary" title={url}>
-    {character === true ? <Icon className="inline-block" path={mdiAccount} size={1} /> : null}
-    &nbsp;
-    {text}
-  </span>
-);
+const CleanMiscLinesRegex = /^(\*|--|~) /sg;
 
-const AnidbDescription = ({ text }) => {
-  const lines = [] as Array<JSX.Element>;
+const CleanMultiEmptyLinesRegex = /\n{2,}/sg;
+
+const LinkRegex = /(?<url>http:\/\/anidb\.net\/(?<type>ch|cr|[feat])(?<id>\d+)) \[(?<text>[^\]]+)]/g;
+
+const AnidbDescription = ({ text }: { text: string | null | undefined }) => {
```

Comment:
You don't need to specify `null | undefined`

Reply:
It is `text: string | null | undefined` because the value passed to it is maybe a string, maybe `null` or maybe `undefined`. It is not `text?: string | null | undefined` because `text` in mandatory.

Context:
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/7761729/2c324076-4012-4ff1-ba38-cc03227d649d)
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/7761729/10161b15-ea87-4fec-a8e7-06e5bce13753)

Reply:
Why pass null or undefined values intentionally? You could have a fallback for the second screenshot -> `?? ''`

Reply:
I forgot about that.

Reply:
Fixed.